### ISSUE TITLE
Missing prefix in branch tracer

### ISSUE THREAD

**Issue by fmontesi** [jolie/jolie#335]
Branch `tracer` should be named `feature/tracer` or `fix/tracer` per the git workflow guidelines.

Also, is the branch still relevant?

**Comment by mwallnoefer** [comment]
Ping? https://github.com/jolie/jolie/tree/tracer

**Comment by mwallnoefer** [comment]
I guess we should remove these un-maintained branches hanging around for years.

**Comment by fmontesi** [comment]
I agree, let's get rid of them. @klag maybe you wanna do a backup? Or has it been integrated already?

Re sourceforge, I'll erase the page or put a redirect to github.
